src/test/ui/proc-macro/invalid-punct-ident-1.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:invalid-punct-ident.rs
+// needs-unwind
```

Comment:
Add a comment here. Suggestion: "if a proc macro panics then the compiler will abort". [This issue](https://github.com/rust-lang/rust/issues/82320) might be worth linking as well.